2026-06-07 18:02:48 UTC

@colourbill-ctrl Can you [fix](https://github.com/InternationalColorConsortium/iccDEV/actions/runs/27100254516/job/79979705593?pr=1258#step:9:238):
```

/home/runner/work/iccDEV/iccDEV/.github/ci/regression/parser-restore-calls.cpp:41:51: warning: enumerated and non-enumerated type in conditional expression [-Wextra]
   41 |       *((icUInt32Number *)pBuf) = read_count == 1 ? icSigEmbeddedProfileType : 0;
      |                                   ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     
```

--- a/.github/ci/regression/parser-restore-calls.cpp
+++ b/.github/ci/regression/parser-restore-calls.cpp
@@ -38,7 +38,7 @@ public:
   {
     ++read_count;
     if (nNum >= sizeof(icUInt32Number)) {
-      *((icUInt32Number *)pBuf) = read_count == 1 ? icSigEmbeddedProfileType : 0;
+      *((icUInt32Number *)pBuf) = read_count == 1 ? (icUInt32Number)icSigEmbeddedProfileType : 0u;
       return sizeof(icUInt32Number);
     }
     return 0;

